public/app/components/fontViewer/fontViewer.js

```diff
@@ -0,0 +1,93 @@
+angular.module('kibibitCodeEditor')
+
+.directive('kbFontViewer', function() {
+  return {
+    scope: {},
+    bindToController: {
+      openFont: '=kbOpenFont'
+    },
+    controller: 'fontViewerController',
+    controllerAs: 'fontViewerCtrl',
+    templateUrl: 'app/components/fontViewer/fontViewerTemplate.html',
+    link: function(scope, element, attrs, fontViewerCtrl) {
+      scope.$watch('fontViewerCtrl.openFont', function(newOpenFont) {
+        console.log('element: ',element);
+        fontViewerCtrl.updateFontView(newOpenFont);
+      });
+    }
+  };
+})
+
+.controller('fontViewerController', [
+  function() {
+    var vm = this;
+
+    vm.fontSize = 100;
+
```

Comment:
`redrawFont` will run twice in this way. you can remove the `ng-change="fontViewerCtrl.redrawFont()"` since it's exactly the same as writing
```javascript
scope.$watch('fontViewerCtrl.openFont', function(newOpenFont) {
  console.log('element: ',element);
  fontViewerCtrl.updateFontView(newOpenFont);
});
```

So, let's keep the `watch` notation for now. `updateFontView` actually updates the font view (instead of the `redrawFont` function :-))